scripts/deploy-kit.js

```diff
@@ -0,0 +1,68 @@
+const deployDAOFactory = require('@aragon/os/scripts/deploy-daofactory.js')
+
+const fs = require('fs');
+const path = require('path');
+const argv = require('yargs').argv
+const namehash = require('eth-ens-namehash').hash
+
+const fileInject = require('./helpers/file_inject.js')
+
+const DAOFactory = artifacts.require('DAOFactory')
+const KreditsKit = artifacts.require('KreditsKit')
+
+const arapp = require('../arapp.json')
+const environment = argv['network'] || argv['environment'] || 'development'
+const apm = arapp.environments[environment].apm
+const ensAddr = arapp.environments[environment].registry || process.env.ENS
+const daoFactoryAddress = arapp.environments[environment].daoFactory || process.env.DAO_FACTORY
+
+
+module.exports = async function(callback) {
+
+  // load networkId; will change with updated truffle
+  const networkPromise = new Promise((resolve, reject) => {
+    web3.version.getNetwork((err, network) => {
+      if (err) {
+        reject(err);
+      } else {
+        resolve(network);
+      }
+    })
+  })
+  const networkId = await networkPromise;
```

Comment:
const networkId = await getNetworkId(web3);

Reply:
Have you tried the new suggestion feature in these comments? It's pretty useful!